tools/compactsnoop.py

```diff
@@ -0,0 +1,389 @@
+#!/usr/bin/python
+# @lint-avoid-python-3-compatibility-imports
+#
+# compactsnoop  Trace compact zone and print details including issuing PID.
+#       For Linux, uses BCC, eBPF.
+#
+# This uses in-kernel eBPF maps to cache process details (PID and comm) by
+# compact zone begin, as well as a starting timestamp for calculating
+# latency.
+#
+# Copyright (c) 2019 Ethercflow
```

Comment:
please use your real name in copyright note.

Reply:
Thanks, I'll fix this.